content/en/docs/setup/production-environment/container-runtimes.md

```diff
@@ -22,6 +22,17 @@ Kubernetes, on Linux:
 - [CRI-O](#cri-o)
 - [Docker](#docker)
 
+{{< note >}}
+Docker container runtime (Dockershim) has been [deprecated](
```

Comment:
This implies that Docker container runtime is deprecated and that dockershim and Docker container runtime are the same.
I suggest more clarity that dockershim is deprecated

Reply:
@PranshuSrivastava would you be willing to revise the language here?

Reply:
@sftim @reylejano sorry I missed this review. I have made the change, PTAL.